packages/terra-form-textarea/src/TextareaField.jsx

```diff
@@ -168,6 +168,7 @@ const TextareaField = (props) => {
         id={inputId}
         onChange={onChange}
         value={value}
+        label={label}
```

Comment:
The label for the TextareaField should be handled by the Field wrapper which creates a native `<label>`. Do we always want to supply both? Making the label required within the base Textarea may force our hand.

Reply:
We want the aria-label on the input to be non-required. In the field composition, we want the label to be required.

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2583/commits/cba49d48d985f385a4e210fc459814560d02c3b2.